packages/@konfik/cli/src/cli.ts

```diff
@@ -177,3 +184,144 @@ export const run = () =>
     provideCwd,
     runMain,
   )
+
+function contextLines(lines: Array<string>): Array<AnsiDoc> {
+  return lines.map((entry) => Doc.text(` ${entry}`))
+}
+
+function changeLines(current: Change & { lines: Array<string> }): Array<AnsiDoc> {
+  return current.lines.map((entry) => {
+    if (current.added) {
+      return pipe(Doc.text(`+${entry}`), Doc.annotate(AnsiStyle.color(AnsiColor.green)))
+    }
+    return pipe(Doc.text(`-${entry}`), Doc.annotate(AnsiStyle.color(AnsiColor.red)))
+  })
+}
+
+function getDiff(oldContents: string, newContents: string) {
```

Comment:
Maybe now is the right time to split up the `cli.ts` file e.g. move the diff related parts into its own file?

Reply:
👍